client/src/components/ui/Select.svelte

```diff
@@ -1,9 +1,12 @@
 <script lang="ts">
+    import { assert } from '../../assert';
     export let options: string[] = [];
-    export let index = 0;
+    export let index: number | null = null;
     export let value = '';
 
-    $: value = options[index];
+    $: {
+        if (index) value = options[index] ?? '';
+    }
```

Comment:
Braces are actually not needed here.
```suggestion
    $: if (index) value = options[index] ?? '';
```